addon/templates/components/power-select/options.hbs

```diff
@@ -5,18 +5,29 @@
 {{/if}}
 {{#each options as |opt index|}}
   {{#if (ember-power-select-is-group opt)}}
-    <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present opt.disabled}} role="option">
-      <span class="ember-power-select-group-name">{{opt.groupName}}</span>
-      {{#component optionsComponent
-        options=(readonly opt.options)
-        select=(readonly select)
-        groupIndex=(concat groupIndex index ".")
-        optionsComponent=(readonly optionsComponent)
-        role="group"
-        class="ember-power-select-options" as |option|}}
-        {{yield option select}}
-      {{/component}}
-    </li>
+    {{#with
```

Comment:
When it comes to documentation, it's tricky but what I think I'd do is:

1. As with all public API options, list it in the `API reference` section.
2. Given that this is a pretty advanced customization (and I imagine you are doing this because you need it), an entry in the cookbook with such example would be great.